src/CattyTests/Bricks/BubbleBrickTests.swift

```diff
@@ -53,7 +53,7 @@ final class BubbleBrickTests: XMLAbstractTest {
         spriteObject.project = project
 
         project.objectList.add(spriteObject)
-        spriteNode.mockedScene = SceneBuilder(project: ProjectMock(width: CGFloat(kIphoneXSceneWidth), andHeight: CGFloat(kIphoneXSceneHeight))).build()
+        spriteNode.mockedScene = StageBuilder(project: ProjectMock(width: CGFloat(kIphoneXSceneWidth), andHeight: CGFloat(kIphoneXSceneHeight))).build()
```

Comment:
Maybe also rename the mockedScene property of the CBSpriteNodeMock.
I would also say that a renaming of the constants kIphoneXSceneWidth etc. should be done.